tests/Fixtures/bookstore/schema.xml

```diff
@@ -400,4 +400,9 @@
         </foreign-key>
     </table>
 
+    <!-- Test custom phpType which is an object type -->
+    <table name="bookstore_custom_test">
+        <column name="id" primaryKey="true" autoIncrement="true" type="INTEGER"/>
+        <column name="test_datatype" type="VARCHAR" required="true" customType="Propel\Tests\Helpers\CustomDatabaseType"/>
```

Comment:
Nitpick: `test_datatype` made me think it's a type not a column. Something like `name="column_with_custom_database_type"`? (except there are too many characters)